src/ActiveInventory.cpp

```diff
@@ -227,17 +222,10 @@ void ActiveInventory::printItems(std::string value) {
     }
 }
 
-
 void ActiveInventory::printHead() {
-    std::cout << 
-    std::left << std::setw(7) << "ID" << 
-    std::left << std::setw(40) << "ITEM" <<
-    std::left << std::setw(17) << "CATEGORY" <<
-    std::left << std::setw(10) << "STOCK" <<
-    std::left << std::setw(15) << "PURCHASE_COST" <<
-    std::left << std::setw(15) << "SALE-PRICE" <<
-    std::left << std::setw(15) << "TAX-ON-ITEM" <<
-    std::left << std::setw(15) << "TOTAL-PRICE" << 
-    std::left << std::setw(10) << "PROFIT" << 
-    std::left << std::setw(10) << "EXPIRATION" << std::endl;
+    std::cout << std::left << std::setw(7) << "ID" << std::left << std::setw(40) << "ITEM" << std::left << std::setw(17)
```

Comment:
clang-format absolutely butchered this.  We can leave it as is for now but if it butchers more stuff like this we may have to play around with our .clang-format file.